metadata_annotation_gdsc.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>This box and the next one should be clarified. Ideally, you want to make making a train/test split just a matter of a few lines with your annotation!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>